namespace-bc-aliases.php

```diff
@@ -19,7 +19,6 @@ class_alias(\Roave\BetterReflection\Reflection\Exception\InvalidDefaultValueType
 class_alias(\Roave\BetterReflection\Reflection\Exception\MethodPrototypeNotFound::class, \BetterReflection\Reflection\Exception\MethodPrototypeNotFound::class);
 class_alias(\Roave\BetterReflection\Reflection\Exception\NoParent::class, \BetterReflection\Reflection\Exception\NoParent::class);
 class_alias(\Roave\BetterReflection\Reflection\Exception\NotAClassReflection::class, \BetterReflection\Reflection\Exception\NotAClassReflection::class);
-class_alias(\Roave\BetterReflection\Reflection\Exception\NotAString::class, \BetterReflection\Reflection\Exception\NotAString::class);
```

Comment:
This is a BC break that needs documentation - also, let's not be scared to call it a 2.0.0.